### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by codecaiine**
I created forms to perform the following functions:

- Creates a Post on behalf of the current_user.
- Create a comment on behalf of the current_user.
I allow Users to add likes to Posts.

### CODE REVIEW

File: app/assets/stylesheets/comments.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Comments controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```
Comment (DanSam5K) [inline app/assets/stylesheets/comments.scss]: - [ ] It is often considered good practice to try and always take out unused files and remove all comments from files

File: spec/helpers/comments_helper_spec.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+# Specs in this file have access to a helper object that includes
+# the CommentsHelper. For example:
+#
+# describe CommentsHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
```
Comment (DanSam5K) [inline spec/helpers/comments_helper_spec.rb]: - [ ] same as above point raised on trailing comments

File: spec/helpers/likes_helper_spec.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+# Specs in this file have access to a helper object that includes
+# the LikesHelper. For example:
+#
+# describe LikesHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
```
Comment (DanSam5K) [inline spec/helpers/likes_helper_spec.rb]: - [ ] same as above

File: config/routes.rb [hunk 4]
```diff
@@ -1,7 +1,12 @@
...
 
-    resources :users, only: %i[index show] do 
-    resources :posts, only: %i[index show]
+  resources :users, only: %i[index show] do 
+    resources :posts, only: %i[index new create show]
+  end
+
+  resources :posts do
+    resources :comments, only: %i[create]
+    resources :likes, only: %i[create]
```
Comment (DanSam5K) [inline config/routes.rb:10]: - [ ] please provide a corresponding routes to get  comments

File: app/controllers/comments_controller.rb [hunk 5]
```diff
@@ -0,0 +1,22 @@
+class CommentsController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.create(text: comment_parameters[:text], author_id: current_user.id, post_id: @post.id)
```
Comment (DanSam5K) [inline app/controllers/comments_controller.rb]: - [ ] I think a comment is created on behalf of the currently logged in user (current_user) of the app. How about making sure the comments controller should have new and create action.

**Review by DanSam5K — CHANGES_REQUESTED** [review]
# Hi @codecaiine, ♻️  Required Changes ♻️
## Highlights 💥 💥 
- [x] Excellent Job on making sure all linter checks pass it is never easy oftentimes 😄  ✔️ 
- [x] I really love the fact you try to write a couple of tests with RSpec ✔️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] It would really make more sense to please provide adequate tests for your project

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>
 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 4 above)
Comment (codecaiine) [reply config/routes.rb:10]: Hi @DanSam5K, 
You can get comments from posts.

File: app/controllers/comments_controller.rb (hunk 5 above)
Comment (codecaiine) [reply app/controllers/comments_controller.rb]: I fixed this.

**Review by dasileker — APPROVED** [review]
## **Hi @codecaiine** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by codecaiine** [comment]
Thank you @dasileker for reviewing and approving my work.

Cheers!

**Pull request merged**